modules/storage-persistent-storage-pv.adoc

```diff
@@ -249,6 +249,8 @@ cluster.
 
 |===
 
+To find the time of the last phase transition, use the field, `LastPhaseTransitionTime`. This field has a timestamp that updates every time a persistent volume (PV) transitions to a different phase (`pv.Status.Phase`).
+
```

Comment:
Maybe add a sample command here? @RomanBednar 
I believe it should look like
`oc get pv <pv-name> -o json | jq '.status.lastPhaseTransitionTime`

Reply:
thx @RomanBednar is the command correct?

Reply:
@gcharot @lpettyjo this example would work as well, but looking at the docs it seems like `jsonpath` output format is more common. It's also slightly more universal as `jq` might not be present on the system. Having an example is good, I  just assumed that getting fields is a common knowledge and we don't need to have it here. The example command would be: `oc get pv <pv-name> -o jsonpath='{.status.lastPhaseTransitionTime}'` 

Real example:
```
$ oc get pv pvc-994591e2-166d-4c53-883b-889f64df1671 -o jsonpath='{.status.lastPhaseTransitionTime}'
2024-11-18T10:26:13Z
```

Reply:
Ah good point @RomanBednar 
@lpettyjo Please use Roman's example instead